src/shape.rs

```diff
@@ -27,22 +27,36 @@ pub trait Shape: Sized {
     /// iterators from complex shapes without cloning.
     ///
     /// [GAT's]: https://github.com/rust-lang/rust/issues/44265
-    fn to_bez_path(&self, tolerance: f64) -> Self::BezPathIter;
+    fn to_path_elements(&self, tolerance: f64) -> Self::PathElementsIter;
```

Comment:
I might have missed this in discussion, but I would not include the `to_` prefix for methods that return iterators.

Reply:
Changed. I've also reworded the documentation a little because it said "Convert to an iterator ..." previously, which is not really accurate.

I changed it to the follwing:
> Returns an iterator over this shape expressed as Bézier path elements.

I'm not _super_ happy with the wording, but it's the best I've got. I'm open to suggestions.

Reply:
Maybe something like,

```
    /// Returns an iterator over the individual ([`PathEl`])s of a [`BezPath`] that
    /// represents this shape.
    ///
    /// This method should not allocate, and is suitable for things like drawing,
    /// as the returned elements map closely to many 2D graphics APIs.
    ///
    /// If you are are more interested in the *segments* of the path (that is, the
    /// constituent lines and curves) then you should use [`Shape::path_segements`]
    /// instead. If you would like to retain the [`BezPath`], use [`Shape::to_path`].
```

In any case I'm happy to touch up the docs in a subsequent PR, I think this is in a good place to merge.

Reply:
I like that, though I'm not sure about the should-not-allocate part since it _does_ allocate for `<BezPath as Shape>::path_elements` (at least until GATs land).

> In any case I'm happy to touch up the docs in a subsequent PR, I think this is in a good place to merge.

Sounds good!